iot_sonoff/models/iot_device.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2018 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import json
+from odoo import api, fields, models
+
+
+class IoTDevice(models.Model):
+    _inherit = 'iot.device'
+
+    is_sonoff = fields.Boolean(store=True, compute='_compute_is_sonoff')
```

Comment:
I would create in the 'iot'  base model a selection field, something like 'iot_type', and extend it here with "sonof". What do you think? This way you wouldn't end with a lot of is_whatever fields.